src/entries/popup/components/Tooltip/Tooltip.tsx

```diff
@@ -85,22 +92,33 @@ export const Tooltip = ({
                       rotate: '45deg',
                       left,
                       right,
+                      ...(arrowDirection === 'up' ? { top: -4 } : {}),
                     }}
                   />
                 </Inline>
                 <Box
+                  style={{
+                    alignItems: 'center',
+                    borderRadius: 6,
+                    display: 'flex',
+                    justifyContent: 'center',
+                    minHeight: 22,
+                  }}
                   background="surfaceSecondaryElevated"
                   padding="7px"
-                  borderRadius="6px"
+                  paddingHorizontal="6px"
                   backdropFilter="blur(26px)"
                 >
-                  <TextOverflow
-                    color={textColor || 'label'}
-                    size={textSize || '16pt'}
-                    weight={textWeight || 'bold'}
-                  >
-                    {text}
-                  </TextOverflow>
+                  <Inline alignVertical="center" space={'6px'}>
+                    <TextOverflow
+                      color={textColor || 'label'}
+                      size={textSize || '16pt'}
+                      weight={textWeight || 'bold'}
+                    >
+                      {text}
+                    </TextOverflow>
+                    {hint ? <ShortcutHint hint={hint} small /> : null}
+                  </Inline>
```

Comment:
any reason to not merge Tooltip and CursorTooltip into 1 component? seems like we don't use Tooltip anymore